tests/integration/test_speakerRecognition_gradient_step_and_eval.py

```diff
@@ -0,0 +1,193 @@
+# ! /usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import os
+import shutil
+import tarfile
+from functools import partial
+from unittest import TestCase
+
+import pytest
+from ruamel.yaml import YAML
+
+import nemo
+import nemo.collections.asr as nemo_asr
+
+logging = nemo.logging
+
+
+@pytest.mark.usefixtures("neural_factory")
+class TestSpeakerRecognitonPytorch(TestCase):
+    manifest_filepath = os.path.abspath(os.path.join(os.path.dirname(__file__), "../data/an4_speaker/train.json"))
+    featurizer_config = {
+        "normalize": "per_feature",
+        "window_size": 0.02,
+        "window_stride": 0.01,
+        "window": "hann",
+        "features": 64,
+        "n_fft": 512,
+        "frame_splicing": 1,
+        "dither": 0.00001,
+        "stft_conv": False,
+    }
```

Comment:
What is the point of this if you have it inside your yaml?

Reply:
Noted